_extras/design.md

```diff
@@ -116,7 +116,117 @@ How do I...
 
 ## Stage 2 - Learning Plan
 
-*To Do: point-form outline of the lesson episodes with time estimates for teaching and exercises.*
+### Introductions and Learning Objectives
+
+*   Teaching: 15 min
+*   Exercises: 0 min (accounted for in teaching time - no separate exercise)
+    *   Ensure setup issues resolved
+    *   Assess how much `dplyr` review will be necessary
+
+### Data Management in R
+
+*   Teaching: 10 min
+*   Exercises: 5 min
+    *   Import Gapminder data (`read.csv()`)
+    *   Gapminder data structure (`str()`, `head()`)
+
+### Data Structures
+
+*   Teaching: 5 min
+*   Exercises: 5 min
+    *   Variables and categories
+    *   Values and replicates
+    *   Absolute values vs. relative values
+
+### `dplyr` Recap
+
+*   Teaching: 10 min
+*   Exercises: 5 min
+    *   `select()`
+    *   `filter()`
+    *   `group_by()`
+    *   `summarize()`
+
+### Scientific Questions and Hypotheses
+
+*   Teaching: 10 min
+*   Exercises: 5 min
+    *   Define: distribution, relationship, comparison, composition
+
+### Introduce 'Choosing a Good Chart'
+
+*   Teaching: 10 min
+*   Exercises: 5 min
+    *   Chart dependends on scientific question and data structure
+    *   "How was the population of Africa distributed across the continent by country in 2007?" 
+
+### `ggplot`: Grammar of Graphics
+
+*   Teaching: 10 min
+*   Exercises: 5 min
+    *   Aesthetics mapping
+    *   Layers
+    *   Themes
```

Comment:
What are the exercises? Changing the aesthetics/layers/themes in an existing plot?